src/components/TheMap.vue

```diff
@@ -72,6 +76,27 @@ export default {
             this.$store.commit("app/updatingData", {
               updatingData: false
             });
+            updateSensorGeoJSON(map);
+            this.$store.watch(
+              ({ timelapse }) => timelapse.sliderVal,
+              () => {
+                updateSensorGeoJSON(map);
+              }
+            );
+            this.$store.watch(
+              ({ app }) => app.updatingData,
+              updatingData => {
+                if (!updatingData) {
+                  updateSensorGeoJSON(map);
+                }
+              }
+            );
+            this.$store.watch(
+              ({ cons }) => cons.layers[1].selected,
```

Comment:
Can we watch `layerSelected` from the `app` store module instead? This would be a minor change but more in line with above where we are watching `updatingData` which comes from the `app` store module. And then you can do `displayInundationLayer(map, layerSelected === 1);`

Reply:
Even better would be refactoring this to:
```
const visibilityStr = layerSelected === 1 ? "visible" : "none"
map.setLayoutProperty("inundation_heat", "visibility", visibilityStr);
```
And remove the `displayInundationLayer`  method.